src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -694,7 +694,7 @@ messageInstD ctxt parentIdent msgIdent messageParts = do
                         , dpTypeE dotProtoFieldType
                         , dpIdentE dotProtoFieldName
                         , HsList (map optionE dotProtoFieldOptions)
-                        , maybeE (HsLit . HsString) dotProtoFieldComment
+                        , HsLit (HsString dotProtoFieldComment)
```

Comment:
Use `str_`?

Reply:
Sounds good; thanks.  Changed in 15327e505a1f8cad11f2e98c8499a0a584a659ec .